jax/experimental/pallas/__init__.py

```diff
@@ -44,11 +44,11 @@
 from jax._src.pallas.utils import when
 
 try:
-  from jax.experimental.pallas import gpu # pytype: disable=import-error
-except (ImportError, ModuleNotFoundError):
+  from jax.experimental.pallas import gpu  # pytype: disable=import-error
+except (ImportError, ModuleNotFoundError, AttributeError):
   pass
 
 try:
-  from jax.experimental.pallas import tpu # pytype: disable=import-error
-except (ImportError, ModuleNotFoundError):
+  from jax.experimental.pallas import tpu  # pytype: disable=import-error
+except (ImportError, ModuleNotFoundError, AttributeError):
```

Comment:
I said this in chat, but I'll copy here: I think catching the AttributeError here is not the right solution. It might make debugging difficult in the future if some change introduces an unexpected AttributeError in the imported module. Rather, the AttributeError should be handled gracefully at the point where the potentially problematic attribute is accessed.

Reply:
Setting `get_compute_capability=None` by default could be better. I made the change. WDYT?

Reply:
Is `None` the correct default value here? What happens further along in the program if this value remains `None`? And do any of the other defined attributes need a default?

Reply:
I'm not familiar with the codebase. Maybe it's better to make a bug report instead of trying to fix.